apps/examples/testcase/le_tc/kernel/tc_libc_unistd.c

```diff
@@ -241,6 +242,10 @@ static void tc_libc_unistd_pipe(void)
 
 	sem_wait(&pipe_sem);
 
+	/* wait for child to exit, and store child's exit status */
+	ret_chk = wait(&status);
```

Comment:
@junmin-kim , I think tx task and rx should be synchronized with sem_wait and sem_post as shown below as we can't assume that which task would get chance to execute first.
What do you say?
```
pipe_tx_func(...)
{
   ...
   tx work
   sem_post(&pipe_sem);
   ...
}

pipe_rx_func(...)
{
   ...
   sem_wait(&pipe_sem);
   rx work
   sem_post(&pipe_sem);
   ...
}
```